Fluent/Post_Porcess.ipynb

```diff
@@ -8,53 +8,97 @@
+      "text/html": [
+       "\n",
+       "        <iframe\n",
+       "            width=\"400\"\n",
+       "            height=\"300\"\n",
+       "            src=\"https://www.youtube.com/embed/SrynvmBNA2E\"\n",
+       "            frameborder=\"0\"\n",
+       "            allowfullscreen\n",
+       "        ></iframe>\n",
+       "        "
+      ],
+      "text/plain": [
+       "<IPython.lib.display.YouTubeVideo at 0x103c68a58>"
+      ]
+     },
+     "execution_count": 1,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "from IPython.display import YouTubeVideo\n",
+    "\n",
+    "YouTubeVideo('SrynvmBNA2E')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<img src=\"./Images/vortices.png\\\" width = \\\"900\\\">\n",
+    "Fig. 3: Animation vorticity contours for flow over stationary cylinder within the CFD Domain for Re =100. \n",
     "\n",
-    "Increasing the rotational speed ($\\alpha$) of the cylinder increases the strength of the generated vortices. This is accompanied by an increase in the thickness of the region with closed streamlines around the cylinder. For vortex shedding to occur, vorticity of large strength must be carried to the outer flow, outside the closed streamlines via diffusion. This region of vorticity, in the outer flow, must be sufficiently larger than area of slowly moving, preferably, recirculating fluid. Under these conditions the vorticity can pile up in a certain region of the flow and vortex shedding can take place. For $\\alpha = 0$, a large region behind the cylinder is associated with slowly moving fluid. The size of this region reduces rapidly with increasing rotational velocity. This explains the disappearance of vortex shedding for $\\alpha = 2.0$ as visualized in Fig. 2.\n",
+    "Increasing the rotational speed (i.e. $\\alpha$) of the cylinder would increase the strength of vorticity generated on the cylinder surface increases. This is accompanied by an increase in the thickness of the region with closed streamlines around the cylinder. For vortex shedding to occur, vorticity of large enough strength must be carried to the outer flow, outside closed streamlines, via diffusion. This region of vorticity, in the outer flow, must also be a sufficiently large area of slowly moving, preferably recirculating fluid. Under these conditions the vorticity can pile up in a certain region of the flow and vortex shedding can take place. For $\\alpha = 0$, a large region behind the cylinder is associated with slowly moving fluid. The size of this region reduces, rapidly, with increase in rotational speed of the cylinder. This explains the disappearance of vortex shedding for $\\alpha = 2$ as visualized in the animation shown in Fig. 3.\n",
```

Comment:
Review this edit of mine carefully. Toward the end link the content of this paragraph to the content of next paragraph to the discussion on plotted streamlines around the cylinder.